index.js

```diff
@@ -0,0 +1,17 @@
+// temp data, will be added to local storage
+const watchedVideos = []
+let watchCount = 0
+
+const watchCountEl = document.getElementById("watch-count-el")
+
+document.addEventListener("click", function(e){
+    if(e.target.id === "watch-count-btn") {
+        addNewVideo()
+    }
+})
+
+function addNewVideo(){
```

Comment:
Could we change the function name to something a little more specific? If I had to guess what the function did based off the name I'd assume it added an actual video. Maybe something like "incrementTotalVideosWatched" or something?

Reply:
refactoring the event listener actually gets rid of this issue, since that was everything that it does so the standalone function is no more :)